modules/grub.sh

```diff
@@ -83,8 +83,11 @@ function grub_install_grub_d_config() {
         run_and_log_cmd "sudo mkdir work/rootfs/boot/efi"
     fi
     run_and_log_cmd "sudo mkdir work/boot/grub-mender-grubenv"
-    run_and_log_cmd "sudo mv work/rootfs/boot/grub/* work/boot/grub-mender-grubenv/"
-    run_and_log_cmd "sudo rmdir work/rootfs/boot/grub"
+    if [ -e work/rootfs/boot/grub ]; then
```

Comment:
I think I vote for keeping this one. In both cases it will fail on the next command, but it will fail in different ways:

* `-e`: Not a directory
* `-d`: Already exists

I think the former is clearer in what the problem is, because already existing is both normal and expected. It's just that we don't want that already existing entry to be a non-directory.